cmd/osbuild-worker/jobimpl-depsolve.go

```diff
@@ -52,7 +52,7 @@ type DepsolveJobImpl struct {
 // (matching map keys).
 func (impl *DepsolveJobImpl) depsolve(packageSets map[string][]rpmmd.PackageSet, modulePlatformID, arch, releasever string) (map[string][]rpmmd.PackageSpec, map[string][]rpmmd.RepoConfig, error) {
 	solver := impl.Solver.NewWithConfig(modulePlatformID, releasever, arch, "")
-	if impl.RepositoryMTLSConfig.Proxy != nil {
+	if impl.RepositoryMTLSConfig != nil && impl.RepositoryMTLSConfig.Proxy != nil {
```

Comment:
```suggestion
	if impl.RepositoryMTLSConfig?.Proxy != nil {
```
Would work as well, right?

Reply:
does that operator exist in go?

Reply:
sorry - mixed it up...

Reply:
ye, js only :>